javascript/organizing-js/es6-modules.md

```diff
@@ -31,8 +31,8 @@ The __node package manager__ is a command-line tool that gives you access to a g
 Read through the npm links below but don't worry about running any of the commands on your computer. This section is about growing your awareness of npm. You will have an opportunity to use what you learn here in upcoming projects.
 
 1. Take a couple minutes to read the [About npm](https://docs.npmjs.com/getting-started/what-is-npm) page - a great introduction to npm.
-2. [This video](https://docs.npmjs.com/getting-started/installing-npm-packages-locally) teaches you how to install packages with npm.
-3. [This tutorial](https://docs.npmjs.com/getting-started/using-a-package.json) covers the `package.json` file, which you can use to manage your project's dependencies.
+2. [This tutorial](https://docs.npmjs.com/downloading-and-installing-packages-locally) teaches you how to install packages with npm.
```

Comment:
The linked article in point 2 does have a video at the end of the tutorial. Nevertheless, I do agree with the wording change from "video" to "tutorial" as that makes more sense.